src/main/resources/lep/custom/xm/xm/entity/queue/system/AcceptSystemEvent$$INDEX_RELOAD$$around.groovy

```diff
@@ -0,0 +1,101 @@
+package lep.custom.xm.xm.entity.queue.system
```

Comment:
Propose to use Groovy class for logic instead of just script. It will allow to prepare Unit test easily
@sergeysenja1992 what do you think?